src/home/index.jsx

```diff
@@ -1,3 +1,9 @@
 import React from 'react'
 
-export const Home = () => <h1>Hermes Web</h1>
+import { Footer } from 'footer/footer'
+import { useStyles } from 'footer/styles'
+
+export const Home = () => <div className={useStyles().divPageHeight}>
+  <div className={useStyles().divHeight}><h1 className={useStyles().h1}>Hermes Web</h1></div>
```

Comment:
Should we have the `<h1/>` tag on a different line? Putting everything on the same line might fool the linter, but not me. :smile:

Reply:
Done